Frontend/src/api/registration/delete/delete_registration.js

```diff
@@ -0,0 +1,20 @@
+export default async function delete_registration(competitor_id, competition_id){
```

Comment:
I think typically the convention in js (and react) is to use camel case. It looks odd in the jsx file to be calling the function `delete_registration` when everything else is camel case. Or is that intentional to emphasize something?

Reply:
not intentional, just me derping when jumping between ruby and js